tests/src/tests.rs

```diff
@@ -402,6 +428,8 @@ fn test(
             for line in part.lines() {
                 compare_ref = get_flag_metadata(line, "Ref").or(compare_ref);
                 validate_hints = get_flag_metadata(line, "Hints").or(validate_hints);
+                validate_autocomplete =
+                    get_flag_metadata(line, "Autocomplete").or(validate_autocomplete);
```

Comment:
Why do we still need this? If there's no `Autocomplete` lines, we just don't validate anything.